### DISCUSSION TITLE
Request to ask for first beginner issue

### DISCUSSION THREAD

**Discussion by sharmaachintya08** [mozilla-mobile/fenix#28798] [General]
I am new to open source but I have around 4 months of experience in android development using kotlin, I've been using mobile firefox for quite some time now and I would like to contribute to this project, but I would need help of the community to get a little boost in this project. Can anyone suggest to me any beginner issues which I can solve and move ahead to higher and more tricky issues and bugs?

**Comment by kbrosnan** [comment]
This is the list of [good first bugs](https://bugzilla.mozilla.org/buglist.cgi?keywords=good-first-bug%2C%20&list_id=16416651&query_format=advanced&resolution=---&keywords_type=allwords&classification=Client%20Software&classification=Components&product=Fenix&product=GeckoView)

To communicate with the team sign up for [Mozilla's Matrix server](https://wiki.mozilla.org/Matrix) and join the `#geckoview` or `#fenix` channels

This repository is going to be archived soon. Work has moved to https://github.com/mozilla-mobile/firefox-android/
